### DISCUSSION TITLE
Led sempre accesi

### DISCUSSION THREAD

**Discussion by djgragra** [indomus/forum#3056] [Hardware & Firmware]
Buongiorno a tutti. Ho collegato questo dimmer per comandare una striscia LED a 220V. Funziona, ma quando spengo in realtà i led rimangono debolmente accesi. Credo si possa risolvere con un condensatore ma non ne sono sicuro e dato che si tratta di 220V sarebbe meglio avere delle sicurezze.
Avete qualche suggerimento in merito?
Grazie
![its-it-okay-to-install-this-dimmer-switch-with-phase-v0-kendtngoiv9c1](https://github.com/indomus/forum/assets/65872226/f8f54042-61da-4b3e-bdfa-ddf34b75af78)

**Comment by Bergoglio** [comment]
Se rimangono debolmente accesi potrebbe essere che gli arriva sempre la fase.
A vedere lo schema elettrico sul dimmer della fotografia il componente dovrebbe proprio tagliare la fase; hai controllato con un multimetro (perché con il caro cercafase non è detto che riesci a vederlo) se sul morsetto L-Out esce qualcosa anche quando è a zero il comando ?
